docs/source/transcoding.md

```diff
@@ -255,7 +255,18 @@ You can run the above command on any number of machines that you would like to d
 
 ## GPU transcoding
 
-When you setup split transcoding for your orchestrator, you can run individual transcoders on not only CPUs, but GPUs as well. The GPUs that transcoders run on can be dedicated purely to transcoding or can also be mining cryptocurrencies simultaneously. See [this guide](https://github.com/mk-livepeer/bot-miner-setup) for instructions on how to setup transcoders on GPUs.
+When you setup split transcoding for your orchestrator, you can run individual transcoders on not only CPUs, but GPUs as well. The GPUs that transcoders run on can be dedicated purely to transcoding or can also be mining cryptocurrencies simultaneously. See [this guide](https://github.com/livepeer/bot-miner-setup) for instructions on how to setup transcoders on GPUs.
+
+**Running a standalone Transcoder with GPUs**
+
+```
+$ livepeer -transcoder -orchAddr <ORCH_SERVICE_URI> -orchSecret <ORCH_SECRET> -nvidia <GPU IDS (comma separated)>
+```
+
+`<ORCH_SECRET>` should be the secret defined by your orchestrator. `<ORCH_SERVICE_URI>` should be the publicly accessible `serviceURI` that your orchestrator registered on-chain.
```

Comment:
Do we need this line as it is a duplicate of a line in the previous "Scaling transcoding" section?